cmd/scaffold.go

```diff
@@ -572,6 +572,7 @@ func (fnb *FlowNodeBuilder) initFvmOptions() {
 	if fnb.RootChainID == flow.Testnet {
 		vmOpts = append(vmOpts,
 			fvm.WithRestrictedDeployment(false),
+			fvm.WithAccountStorageLimit(false), // XXX make this configurable, should be true by default
```

Comment:
@jordanschalm are you happy with keeping this `false` for testnet?

Reply:
I don't think it is necessary to set this. This is fixed by lowering the token supply (https://github.com/onflow/flow-go/pull/743/commits/38ef15178a40b1d4eb97e8e18c86aac07615a9bb)